setup.py

```diff
@@ -16,7 +16,7 @@
         "pytest-rerunfailures>=10.2,<11",
         "eth-tester[py-evm]>=0.6.0b6,<0.7",
         "py-evm>=0.5.0a3,<0.6",
-        "web3==5.27.0",
+        "web3==5.31.3",
```

Comment:
Can probably keep it like this just for occasionally testing updates for v6 until it's released
```suggestion
        "web3>=6.0.0b10,<7.0.0",
```